community.md

```diff
@@ -49,6 +49,7 @@ It would be great to hear from you especially if you have any of the above and w
 
 | Event name and description                                          | Speaker      | Location | Date        |
 |---------------------------------------------------------------------|--------------|----------|-------------|
+| Knative, OpenFaaS, Kubeless : la guerre des frameworks serverless est lancée ! | Laurent Grangeau | Lyon, France | 24-May-2019 |
```

Comment:
Looks like the URL is: https://mixitconf.org/

Reply:
Ok added it though the events for 2019 are not yet public